tests/specs/permission/ignore_env/flags/__test__.jsonc

```diff
@@ -0,0 +1,21 @@
+{
+  "envs": {
+    "VAR1": "1",
+    "VAR2": "2"
+  },
+  "tests": {
+    "all": {
+      "args": "run --ignore-env main.ts",
+      "output": "undefined\nundefined\n[Object: null prototype] {}\n"
+    },
+    "some": {
+      "args": "run --ignore-env=VAR1 main.ts",
+      "output": "undefined\nerror: [WILDCARD]\n",
+      "exitCode": 1
+    },
+    "some_allow_env": {
+      "args": "run --ignore-env=VAR1 --allow-env main.ts",
+      "output": "undefined\n2\n[WILDCARD]"
+    }
+  }
```

Comment:
Can you please add `--ignore-env` and `--deny-env` interaction here? And preferably all three too